try:
    import main
except Exception as x:
    input(x)
